utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -37,17 +45,23 @@ class HtmlParser private constructor(
     }
 
     if (htmlContent.contains(CUSTOM_IMG_TAG)) {
-      htmlContent = htmlContent.replace(CUSTOM_IMG_TAG, REPLACE_IMG_TAG, /* ignoreCase= */false)
-      htmlContent = htmlContent.replace(
-        CUSTOM_IMG_FILE_PATH_ATTRIBUTE,
-        REPLACE_IMG_FILE_PATH_ATTRIBUTE, /* ignoreCase= */false
-      )
+      htmlContent = htmlContent.replace(CUSTOM_IMG_TAG, REPLACE_IMG_TAG)
+      htmlContent = htmlContent.replace(CUSTOM_IMG_FILE_PATH_ATTRIBUTE, REPLACE_IMG_FILE_PATH_ATTRIBUTE)
       htmlContent = htmlContent.replace("&amp;quot;", "")
     }
 
+    // https://stackoverflow.com/a/8662457
+    if (supportsLinks) {
+      htmlContentTextView.movementMethod = LinkMovementMethod.getInstance()
+    }
+
     val imageGetter = urlImageParserFactory.create(htmlContentTextView, entityType, entityId, imageCenterAlign)
 
-    val htmlSpannable = HtmlCompat.fromHtml(htmlContent, HtmlCompat.FROM_HTML_MODE_LEGACY, imageGetter, LiTagHandler()) as Spannable
+    val htmlSpannable = CustomHtmlContentHandler.fromHtml(
```

Comment:
Earlier LiTagHandler was getting used to modify the bullets but now it is not getting used. Not sure how to introduce this back in the implementation. Need suggestions from @BenHenning

Reply:
Could we create another custom tag handler for ``li`` tags? I'm wondering if we can leverage the existing tag handler machinery to also make the list tag adjustment work correctly.